client/lib/invites/stores/invites-sent.js

```diff
@@ -9,7 +8,7 @@ import { reducer, initialState } from 'lib/invites/reducers/invites-sent';
 
 const InvitesSentStore = createReducerStore( reducer, initialState );
 
-InvitesSentStore.getSuccess = formId => InvitesSentStore.get().getIn( [ 'success', formId ] );
-InvitesSentStore.getErrors = formId => InvitesSentStore.get().getIn( [ 'error', formId ] );
+InvitesSentStore.getSuccess = formId => InvitesSentStore.get().success[ formId ];
+InvitesSentStore.getErrors = formId => InvitesSentStore.get().error[ formId ];
```

Comment:
I haven't actually tested yet, but the key in the state reducer is `errors`.

Also the key for success is `success`.  Should it be plural or singlular? Lets make both error/success consistent!

Reply:
thanks, good catch!